### ISSUE TITLE
retire this blueprint

### ISSUE THREAD

**Issue by andrewmallis** [kalamuna/hot_boilerplate#2]
@reynoldsalec @pirog @soniktrooth – suggest we delete this project. Thoughts.

**Comment by reynoldsalec** [comment]
Agreed @andrewmallis, we'd want to tear this down a bit, a lot of the metadata that's referenced in these files just isn't necessary with the way we use HotApps! right now.

**Comment by andrewmallis** [comment]
Once we tear it down what's left?
Does the apps infrastructure even work?
Are we reverting to straight up features?

**Comment by reynoldsalec** [comment]
Reverting to straight up features. There really isn't a reason for us to use apps, the overhead of the app server isn't really worth the time it saves us.

Looking at things, I really don't see much here that we'll want. Rolling straight-up features seems best.
